.gitlab-ci.yml

```diff
@@ -35,14 +34,6 @@ variables:
     paths:
       - artifacts/
 
-.determine_version:                &determine_version
-  - VERSION="$(sed -r -n '1,/^version/s/^version = "([^"]+)".*$/\1/p' Cargo.toml)"
```

Comment:
Why is this removed?

Reply:
was used only in `publish-snap.sh` so moved there.
Is there a reason for it to stay here?

Reply:
used only for publish snap